Environment_Engine/Environment_Engine.csproj

```diff
@@ -139,6 +139,7 @@
     <Compile Include="Query\Absorptance.cs" />
     <Compile Include="Query\BottomLeft.cs" />
     <Compile Include="Query\BottomRight.cs" />
+    <Compile Include="Query\GroupingPanels.cs" />
```

Comment:
File name should be `Group`

Reply:
File name here needs to be `Group` too - so probably best to compile and recommit the CSProj changes :smile: